client/app/main/student/student.controller.js

```diff
@@ -14,6 +14,16 @@ function StudentCtrl($scope, $state, $stateParams, Student, toastr, Modal) {
     $scope.student = result.student;
     $scope.interventions = result.interventions;
     $scope.outreaches = result.outreaches;
+    Student.list({id: $stateParams.id}, function(data) {
+      var record = data.currentRecord.entries;
+      var enrolled = 0;
+      var present = 0;
+      _.forEach(record, function(entry) {
```

Comment:
`(entry.enrolled / entry.present).toFixed(2)` returns a decimal value. Unless you can think of a better way to do this, I'm using `Math.floor((entry.present / entry.enrolled) * 100)` so the output is an integer.